test/OrchardCore.Tests/Data/JsonDynamicTests.cs

```diff
@@ -337,4 +341,479 @@ public void JsonDynamicValueMustConvertToUri()
 
         Assert.Equal(expectedValue, (Uri)myDynamic);
     }
+
+    [Fact]
+    public void JsonDynamicValueCanImplicitlyConvertToBool()
+    {
+        var expectedValue = true;
+        dynamic myDynamic = new JsonDynamicValue(JsonValue.Create(expectedValue));
+
+        Assert.True(expectedValue == myDynamic);
+        Assert.False(expectedValue != myDynamic);
```

Comment:
I don't think we need the second check while it's already checked before

Reply:
No, both operators `==, !=` are implemented on `JsonDynamicValue` and should be tested. But I agree that these tests are not the most beautiful.

Reply:
I see, so ignore my above comment